### DISCUSSION TITLE
How to get texture id from Texture in filament

### DISCUSSION THREAD

**Discussion by dandingol03** [google/filament#4645] [Q&A] [answered]
HI,
  Here is the thing. I want to use the ImGui::Image to show my texture image in filament viewer runtime. However i need to pass texture id to it . I check the ``Texture`` class in filament, there is no api related to texture id . So i come here for help~

**Comment by dandingol03** [comment, marked as answer]
done

**Reply by zhangbo-0213** [reply]
Hi,I have the same question, Could  you please tell how handle this? Thanks
